CNAME

```diff
@@ -1 +0,0 @@
-www.gradleup.com
```

Comment:
Was that needed for DNS verification maybe? Ping @deepy ?

Reply:
Yeah we need to re-add it before/after merge, it's used by GH to map the pages to the domain
We don't need to worry about the verification as the domain is verified by the GH organization, but I think it's omitted here to make the preview easier

Reply:
It is no longer needed for modern GitHub Pages. It was a culprit for static sites long ago